nixos/modules/services/misc/readarr.nix

```
@@ -0,0 +1,89 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.readarr;
+in
+{
+  options = {
+    services.readarr = {
+      enable = mkEnableOption (lib.mdDoc "Readarr");
+
+      dataDir = mkOption {
+        type = types.str;
+        default = "/var/lib/readarr/.config/Readarr";
```

```suggestion
        default = "/var/lib/readarr/";
```
This would be more like other services work

---

Again all the other *arr modules/packages in nixpkgs follow this standard. Should we change it for all of them?

---

I imagine that might break existing installations, ie. they wouldn't find their config. So might be low value to fix up there. But if it works here with the shorter path, no reason not to use it.

---

I switched to @SuperSandro2000 suggestion as this is a new module. Let's go with good practices.